sys/kern/ringbuf.c

```diff
@@ -10,29 +10,31 @@ void ringbuf_init(ringbuf_t *rb, void *buf, size_t size) {
   rb->data = buf;
 }
 
-static void produce(ringbuf_t *buf, unsigned bytes) {
-  assert(buf->count + bytes <= buf->size);
-  assert(buf->head + bytes <= buf->size);
+/* below two function are to help to track ringbuf pointers in case when it is
```

Comment:
```suggestion
/* below two functions are to help to track ringbuf pointers in case when it is
```
```suggestion
/* below two function are to help to track ringbuf pointers in case when it is
```